drivers/bus/acpi_new/uacpiosl.c

```diff
@@ -0,0 +1,389 @@
+#include <acpi.h>
+#include <kernel_api.h>
+#include <uacpi/uacpi.h>
+#include <uacpi/event.h>
+
+
+#include <initguid.h>
+#include <ntddk.h>
+#include <ntifs.h>
+#include <mountdev.h>
+#include <mountmgr.h>
+#include <ketypes.h>
+#include <iotypes.h>
+#include <rtlfuncs.h>
+#include <arc/arc.h>
```

Comment:
Alternatively you can move the inclusions to the proper precompiled header file.